src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/BasicHtmlWebResponseObject.Common.cs

```diff
@@ -244,4 +221,25 @@ private static void ParseAttributes(string outerHtml, PSObject elementObject)
 
         #endregion Methods
     }
+
+    internal static class HtmlParser
```

Comment:
Can it be private class in BasicHtmlWebResponseObject?

And I'd add a comment why we need the helper class.